Pirate/Program.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+
+namespace Pirate
+{
+    class Program
+    {
+
+        static string GetUserResponse(string question)
+        {
+            Console.WriteLine(question);
+            string userResponse = Console.ReadLine();
+            while (userResponse == "")
+            {
+                Console.WriteLine("Please enter yee a response.");
+                Console.WriteLine(question);
+                userResponse = Console.ReadLine();
+            }
+            return userResponse;
+        }
+
+
+        static void Main(string[] args)
+        {
+
+
+            Console.WriteLine("ARRRR Welcome YEE to my Pirate Adventure!");
+            Console.WriteLine("");
+            string name = GetUserResponse("AAR what is your PIRATE name?");
+            Console.Clear();
+            string lastname = GetUserResponse("And what is yee Pirate last name?");
+            Console.Clear();
+            string newpiratename = (name + " " + lastname);
+
+            Console.WriteLine($"Welcome {newpiratename}, To my pirate adventure!");
+
+            string answer;
+
+            do
+
+            {
+
+                answer = GetUserResponse("Would you like to go on an adventure? Enter yes or no");
+                Console.Clear();
+
+                if (answer == "no")
```

Comment:
This if-statement, in the context of the loop that it's within, could probably be re-structured to make the user experience a little better.  The program is written so that the user can't continue unless they answer "yes", so we don't really need to check specifically for "no".  We could do:
`if (answer == "yes") { Console.WriteLine("PEERFECT LET'S BEGIN!!"); } else { Console.WriteLine("That's an unnaceptable answer me matey!"); }`
This way, we don't care what they answered if it wasn't "yes", because anything other than "yes" is an unacceptable answer.
As it is currently, if they answer "No" (with a capital N), they aren't given any feedback, only re-prompted with the question.